blog/electron-17-0.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Electron 17.0.0
+date: 2022-02-01T00:00:00.000Z
+authors:
+    - name: mlaurencin
+      url: 'https://github.com/mlaurencin'
+      image_url: 'https://github.com/mlaurencin.png?size=96'
+    - name: VerteDinde
+      url: 'https://github.com/VerteDinde'
+      image_url: 'https://github.com/VerteDinde.png?size=96'
+slug: electron-17-0
+
+---
+
+Electron 17.0.0 has been released! It includes upgrades to Chromium `98`, V8 `9.8`, and Node.js `16.13.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 17.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### Electron Release Cadence Change
+
+As of Electron 15, Electron will release a new major stable version every 8 weeks. You can read the [full details here](https://www.electronjs.org/blog/8-week-cadence).
+
+Additionally, Electron has changed supported versions from latest three versions to latest four versions until May 2022. [See our versioning document](https://www.electronjs.org/docs/latest/tutorial/electron-versioning) for more detailed information about versioning in Electron. After May 2022, we will return to supporting latest three versions.
+
+### Stack Changes
+
+* Chromium `98`
+    * [New in Chrome 98](https://developer.chrome.com/blog/new-in-chrome-98/)
+* Node.js `16.13.0`
+    * [Node 16.13.0 blog post](https://nodejs.org/en/blog/release/v16.13.0/)
+* V8 `9.8`
+    * [V8 9.8 blog post](https://v8.dev/blog/v8-release-98)
```

Comment:
let's remove this link since it doesn't exist and we can add it back if it publishes.